client/allocrunner/taskrunner/task_runner_hooks.go

```diff
@@ -390,6 +412,7 @@ func (tr *TaskRunner) updateHooks() {
 		// Run the update hook
 		var resp interfaces.TaskUpdateResponse
 		if err := upd.Update(tr.killCtx, &req, &resp); err != nil {
+			tr.emitHookError(err, name)
 			tr.logger.Error("update hook failed", "name", name, "error", err)
```

Comment:
Not your PR - but I'm noticing that hook errors are treated inconsistently (e.g. aggregation and returns vs logging and moving on) - would be nice to either make them consistent or note why they are the way they are.  Given that the errors are informational to users, I assume it's not a bit deal to display them anyway.